### PR TITLE
[NUI] Clean up SubWindowTest.cs

### PR DESCRIPTION
**Pull request by dongsug-song**
### Description of Change ###
[NUI] Clean up SubWindowTest.cs
- change testcase methods' names.
- change `case`'s variable as constant.

### API Changes ###
none

### CODE REVIEW

File: test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/SubWindowTest.cs [hunk 1]
```diff
@@ -109,144 +149,142 @@ public void WinTouchEvent(object sender, Window.TouchEventArgs e)
...
-            tm = new Timer(3000);
-            tm.Tick += Tm_Tick;
-            tm.Start();
+            timer = new Timer(3 * oneSecond);
+            timer.Tick += TimerTick;
+            timer.Start();
         }
 
-        private bool Tm_Tick(object source, Timer.TickEventArgs e)
+        private bool TimerTick(object source, Timer.TickEventArgs e)
```
Comment (everLEEst) [inline test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/SubWindowTest.cs:262]: I suggest name DisposeTimer or DisposeSubWinSecondTimer to know what this timer should be supposed to do.

File: test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/SubWindowTest.cs [hunk 2]
```diff
@@ -1,102 +1,142 @@
...
+                PointSize = testPointSize,
+                HeightResizePolicy = ResizePolicyType.FillToParent,
+                WidthResizePolicy = ResizePolicyType.FillToParent
+            };
             mainWin.Add(text);
 
-            Animation animation = new Animation(2000);
-            animation.AnimateTo(text, "Orientation", new Rotation(new Radian(new Degree(180.0f)), PositionAxis.X), 0, 500);
-            animation.AnimateTo(text, "Orientation", new Rotation(new Radian(new Degree(0.0f)), PositionAxis.X), 500, 1000);
+            Animation animation = new Animation(2 * oneSecond);
```
Comment (everLEEst) [inline test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/SubWindowTest.cs:75]: animation is meaningless variable name.
what about testRotationAnim?

File: test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/SubWindowTest.cs [hunk 3]
```diff
@@ -1,102 +1,142 @@
...
             belowParent = false;
 
-            TextLabel text = new TextLabel("Hello Tizen NUI World");
-            text.HorizontalAlignment = HorizontalAlignment.Center;
-            text.VerticalAlignment = VerticalAlignment.Center;
-            text.TextColor = Color.Blue;
-            text.PointSize = 12.0f;
-            text.HeightResizePolicy = ResizePolicyType.FillToParent;
-            text.WidthResizePolicy = ResizePolicyType.FillToParent;
+            TextLabel text = new TextLabel("Hello Tizen NUI World")
```
Comment (everLEEst) [inline test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/SubWindowTest.cs:64]: I suggest name of this variable to testText, helloWorldLabel.

File: test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/SubWindowTest.cs [hunk 4]
```diff
@@ -1,102 +1,142 @@
...
+                log.Debug(tag, $"subWinThird is already created");
                 return;
             }
-            subWin3 = new Window("subWin3", new Rectangle(0, 0, 300, 300), false);
-            subWin3.BackgroundColor = Color.Blue;
+            subWinThird = new Window("subWinThird", new Rectangle(0, 0, 3 * testSize, 3 * testSize), false)
+            {
+                BackgroundColor = Color.Blue
+            };
             View dummy = new View()
```
Comment (everLEEst) [inline test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/SubWindowTest.cs:95]: what about subWinThirdChildView? or simply ChildDummyView, ChildPlaceholder?

File: test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/SubWindowTest.cs [hunk 5]
```diff
@@ -1,102 +1,142 @@
...
         {
             if (e.Key.State == Key.StateType.Down)
             {
-                log.Fatal(tag, $"key down! key={e.Key.KeyPressedName}");
+                log.Debug(tag, $"key down! key={e.Key.KeyPressedName}");
 
                 switch (e.Key.KeyPressedName)
                 {
-                    case "6":
+                    case key6:
```
Comment (everLEEst) [inline test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/SubWindowTest.cs:125]: if key 6, 7 means parentabove, below,
what about name them, KeyParentAbove, KeyParentBelow?
also,
Key is constant variable,
so I personally prefer to name constant with all upper case like,
KEY_PARENT_ABOVE
KEY_PARENT_BELOW
KEY_NUM_1
KEY_ESCAPE
..

File: test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/SubWindowTest.cs [hunk 6]
```diff
@@ -109,144 +149,142 @@ public void WinTouchEvent(object sender, Window.TouchEventArgs e)
...
         {
-            log.Fatal(tag, "test 1 : 1) make sub window-1 2) add some dummy object");
-
-            subWin1 = new Window("subwin1", new Rectangle(500, 500, 300, 300), false);
-            subWin1.BackgroundColor = Color.Blue;
+            subWinFirst = new Window("subWinFirst", new Rectangle(5 * testPosition, 5 * testPosition, 3 * testSize, 3 * testSize), false)
+            {
+                BackgroundColor = Color.Blue
+            };
             View dummy = new View()
```
Comment (everLEEst) [inline test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/SubWindowTest.cs:206]: ChildView, SubWinFisrtChildView...

File: test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/SubWindowTest.cs [hunk 7]
```diff
@@ -109,144 +149,142 @@ public void WinTouchEvent(object sender, Window.TouchEventArgs e)
...
         {
-            log.Fatal(tag, $"test 3 : 1) create sub window2 which doesn't have any setting 2) dispose it after 3 second delay");
-            subWin2 = null;
-            subWin2 = new Window();
-            tm = new Timer(3000);
-            tm.Tick += Tm_Tick;
-            tm.Start();
+            subWinSecond = null;
+            subWinSecond = new Window();
+            timer = new Timer(3 * oneSecond);
```
Comment (everLEEst) [inline test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/SubWindowTest.cs:225]: not only the timer tick fuction name,
this timer variable name would be better to change disposeTimer or disposeSubWinSecondTimer

**Review by everLEEst — COMMENTED** [review]
more comments about variable names.

File: test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/SubWindowTest.cs (hunk 1 above)
Comment (dongsug-song) [reply test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/SubWindowTest.cs]: Thank for review. I fixed it.

File: test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/SubWindowTest.cs (hunk 2 above)
Comment (dongsug-song) [reply test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/SubWindowTest.cs]: Thank for review. I fixed it.

File: test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/SubWindowTest.cs (hunk 3 above)
Comment (dongsug-song) [reply test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/SubWindowTest.cs]: Thank for review. I fixed it.

File: test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/SubWindowTest.cs (hunk 4 above)
Comment (dongsug-song) [reply test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/SubWindowTest.cs]: Thank for review. I fixed it.

File: test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/SubWindowTest.cs (hunk 5 above)
Comment (dongsug-song) [reply test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/SubWindowTest.cs]: Thank for review. I fixed it.

File: test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/SubWindowTest.cs (hunk 6 above)
Comment (dongsug-song) [reply test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/SubWindowTest.cs]: Thank for review. I fixed it.

File: test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/SubWindowTest.cs (hunk 7 above)
Comment (dongsug-song) [reply test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/SubWindowTest.cs]: Thank for review. I fixed it.

**Review by Seoyeon2Kim — APPROVED** [review]

**Review by jaehyun0cho — APPROVED** [review]
LGTM

**Review by everLEEst — APPROVED** [review]
Looks good to me :)
thank you for accepting comments!

**Pull request merged**